libs/common/src/constants/fee-record-status.ts

```diff
@@ -5,4 +5,5 @@ export const FEE_RECORD_STATUS = {
   READY_TO_KEY: 'READY_TO_KEY',
   RECONCILED: 'RECONCILED',
   PENDING_CORRECTION: 'PENDING_CORRECTION',
+  TO_DO_AMENDED: 'TO_DO_AMENDED',
```

Comment:
just wondering, would `AMENDED_AND_TO_DO` be better? opinions? tbh sound a bit naff (my suggestion)

Reply:
I think I prefer `TO_DO_AMENDED`